.github/actions/phar/action.yaml

```diff
@@ -7,6 +7,10 @@ runs:
     - name: Install tools
       uses: ./.github/actions/tools
 
+    - name: Launch Castor to generate stubs
```

Comment:
I would remove the stubs from the phar actually

Reply:
Done